src/tactics.ml

```diff
@@ -103,6 +103,41 @@ let handle_tactic : sig_state -> Proof.t -> p_tactic -> Proof.t =
       handle_refine (Rewrite.reflexivity tac.pos ps)
   | P_tac_sym           ->
       handle_refine (Rewrite.symmetry tac.pos ps)
+  | P_tac_why3(s)    ->
+      (* get the goal to prove *)
```

Comment:
Please, capitalize the comments, and add a dot a the end of the sentence.

Reply:
Fixed in 9e1eb52.